srcpkgs/espanso/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'espanso'
+pkgname=espanso
+version=2.1.6beta
+revision=1
+archs="x86_64 aarch64"
```

Comment:
remove this

Reply:
It seems that the authors only support 64-bit architectures. It won't compile for non-64-bit architectures.

Reply:
Do you have a link about that? It's a high level tool, so I don't really know why this would be necessary. Anyway, there are also other 64bit architectures, most notably -musl versions and ppc64le.